packages/docs/src/pages/sx-prop.mdx

```diff
@@ -117,3 +117,22 @@ export default props => (
   />
 )
 ```
+
+## Using the `sx` Prop in MDX
+
+In order to render your Markdown components, MDX uses its own
+custom pragma. Eventually we'd like to make it more seamless to
+use `sx` in MDX files, but for now you'll have to move all components
+that accept the prop to their own file.
+
+```js
+// src/my-sx-components.js
+/** @jsx jsx */
+import { jsx, Box as BaseBox, Flex as BaseFlex } from 'theme-ui'
+
+export const Box = props => <BaseBox {...props} />
+export const Flex = props => <BaseFlex {...props} />
```

Comment:
I feel like this example looks more complex than it needs to to show what's happening – instead of using `Box` and `Flex`, I think we could have something like:

```jsx
// src/mdx-component.js
/** @jsx jsx */
import { jsx } from 'theme-ui'

export const Div = props => <div {...props} />
```